resources/views/components/airline/pagination.blade.php

```diff
@@ -0,0 +1,4 @@
+@props(['airlines'])
```

Comment:
You've got one pagination component for each of the modules, which are 2 at the moment, and the only difference between them is that one receives a prop names `airlines` and the other a prop named `cities`. For cases like this, you should have a single pagination component that receives a prop such as `data`, which can be either a `airlines` collection or a `cities` collection (which, btw, doesn't matter which one it is), and renders `$data->links()`. Always remember these things so you don't end up with more files/code than you should.